content/blog/2021/cff-best-practises.md

```diff
@@ -0,0 +1,71 @@
+
+2. **Just like variable names, titles are important** 
+
+Give your talk a name that accurately describes the focus of that session in regards to potential attendees. Title is going to be the one line elevator pitch for your talk. This is going to be your first impression for both selection committee and audience. Your title will be listed with hundreds of proposals from various speakers when selection committee goes through them, even though selection doesn't solely depend on the title of the talk but remember, first impression is the best impression. Your title will be also listed along with all the talks on the conference day for the audience to go through. They are going to decide which track to attend based on your title. Some of them from last year being, 
+
+1. Under and Dunder - Python secret functions
+2. Taking a peek under the hood - Interpreting black box models
+3. How I built a Ray Tracer in Pure Python
+4. So, You Want to Build an Anti-Virus Engine?
+5. Developing a Single-Sign-On Service using Django
+6. Logging the right way!
+7. Meme Saheb - Using Dank Learning to Generate Original Meme Captions
+
+3. **Documentation is important, so is the Description** 
+
+A brief description of the talk is needed here. Do provide as much detail as you can to guide us through your plans to cover the topic. The talk duration needs to be 30 minutes. Hence, a rough timeline detailing how you will be spending time explaining the concept would be a great idea to start with. Keep your description elaborate. It should state the problem, how you are trying to solve the problem and what are the take away for the audience. Remember, description is the part where you can convince the selection team that you know what you are talking about.
+
+Also, to make it better you can add an outline section at end of the description. Outline should list out the topics you cover in the talk and roughly how much time you are going to take to deliver them.
+
+4. **Getting the Audience Level right**
+
+At what level of Python and other topic-specific experience or expertise is the talk aiming for? We realize this may be too narrow for a talk, for example, if the talk is aimed at experienced programmers but novices to Python, but give it your best shot. Reviewers need to know what level of Python experience is targeted and also what level(s) of domain-specific expertise is targeted, for example, networking, SQL, database, etc. Please also add references or requirements if any in regards to your talk that attendees need to take care of before joining in.
+
+5. **Propose early and be worry free!**
+
+This year, we will be starting the review process as the proposals come in, and not at the end. Proposals submitted early will get more attention from the reviewers who would be able to provide you the right feedback. 
```

Comment:
I'm not sure if this is accurate. 
Isn't the review starting after the CFP closes?CC: @laisha-wadhwa @raukadah

Reply:
Even if it is the latter, doesn't matter. I think best practice should always be proposing early because then it helps mentorship organize more quickly. Resolving.